src/JDA/CoreBundle/Resources/views/Widget/widget.html.twig

```diff
@@ -106,7 +106,7 @@
 				<div id="description">
 			
 				<div class="widget-textbox ingest-info">
-				  <textarea id="widget-description" ></textarea>
+				  <textarea id="widget-description" > </textarea>
```

Comment:
Sorry, got confused with my testing files. I updated it again. "Placeholder" should get rid of "None" in Description field.